libkbfs/data_types.go

```diff
@@ -640,6 +641,28 @@ func (et EntryType) String() string {
 	return "<invalid EntryType>"
 }
 
+// EXCL indicates whether O_EXCL is set on a fuse call
+type EXCL uint32
+
+const (
+	// NoEXCL indicates O_EXCL is not set
+	NoEXCL EXCL = 0
+
+	// WithEXCL indicates O_EXCL is set
+	WithEXCL EXCL = syscall.O_EXCL
```

Comment:
Got you. I'm in for making a simple function to make EXCL from fuse/dokan calls. But since it's still the O_EXCL semantics, do you think we can keep the value to align with `O_EXCL` on Linux?